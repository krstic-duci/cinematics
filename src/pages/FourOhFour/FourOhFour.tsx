const FourOhFour = () => <div>Not found</div>;

export default FourOhFour;
