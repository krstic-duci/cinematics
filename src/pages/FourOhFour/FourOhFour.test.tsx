import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

import FourOhFour from "./FourOhFour";

describe("FourOhFour component", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(
      <MemoryRouter>
        <FourOhFour />
      </MemoryRouter>
    );
    expect(tree).toMatchSnapshot();
  });
});
