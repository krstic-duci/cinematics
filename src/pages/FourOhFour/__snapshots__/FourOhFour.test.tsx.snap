// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour component should match snapshot 1`] = `
<div
  className="text-center"
>
  No movies here, try some other page please...
</div>
`;
